python/lsst/cp/pipe/cpFlatNormTask.py

```diff
@@ -294,15 +294,17 @@ def run(self, inputMDs, inputDims, camera):
 
         outputScales = defaultdict(lambda: defaultdict(lambda: defaultdict(lambda: defaultdict(float))))
 
+        # Note that the enumerated "detId"/"expId" here index the
+        # "detScales" and "expScales" arrays.
```

Comment:
This comment does enhance clarity, but I'd prefer not reusing `detId` and `expId` here since they were defined previously (lines 254-255 or lines 273-274). Perhaps `for detectorIdx, detector in enumerate(expSet)` and `for exposureIdx, exposure in enumerate(expSet)`?

Reply:
I've changed every variable in the `run` method that is a matrix index to have the `Idx` suffix, to avoid confusion with dataId identifiers.